libraries/ui/src/Card.tsx

```diff
@@ -15,37 +21,61 @@ export const Card: React.FC<CardProps> = ({
   imageSrc,
   title,
   subtitle,
+  ctaMetadata,
   ctaUrl,
+  ctaText,
+  isEntireCardClickable = false,
   isExternalUrl = false,
   className = '',
   imageClassName = '',
 }) => {
+  const Wrapper = isEntireCardClickable && ctaUrl ? 'a' : 'div';
+
   return (
-    <div className={`card flex flex-col items-start ${className}`}>
-      <div className="card__image-container w-[323px] h-[223px] mb-3">
+    <Wrapper
+      {...(isEntireCardClickable && ctaUrl && { href: ctaUrl, ...newTabProps(isExternalUrl) })}
```

Comment:
i think id prefer multiple `isEntireCardClickable` checks for readability:
```suggestion
      href={isEntireCardClickable ? ctaUrl : undefined}
      {isEntireCardClickable ? ...externalUrlProps()}
```